src/raps/unlockAndSwap.js

```diff
@@ -15,60 +13,42 @@ import { createNewAction, createNewRap, RapActionTypes } from './common';
 export const estimateUnlockAndSwap = async ({
   inputAmount,
   inputCurrency,
-  inputReserve,
   outputAmount,
   outputCurrency,
-  outputReserve,
+  tradeDetails,
 }) => {
   if (!inputAmount) inputAmount = 1;
   if (!outputAmount) outputAmount = 1;
 
   const isValid = isValidSwapInput({
     inputCurrency,
-    inputReserve,
     outputCurrency,
-    outputReserve,
   });
 
   if (!isValid) return ethUnits.basic_swap;
```

Comment:
should we increase this number? AFAIK some specific swaps are more expensive on v2

Reply:
very good point!